Ghidra/Processors/MIPS/data/languages/mips32Instructions.sinc

```diff
@@ -219,7 +219,11 @@ define pcodeop special2;
 # 0111 11ss ssst tttt mmmm mLLL LL00 0100
 :ins RT, RSsrc, lsb, InsSize    is $(AMODE) & prime=0x1F & fct=0x04 & RT & RTsrc & RSsrc & lsb & msbd & InsSize {
     tmpa:$(REGSIZE) = -1;
-    tmpa = tmpa >> ($(REGSIZE) - InsSize);
+@if REGSIZE == "4"
```

Comment:
$(REGSIZE)*8 might be cleaner

Reply:
> Note: I considered to use `$(REGSIZE) * 8` but the instruction above uses `@if` so I did it in the same style.

For some reason `:ext` uses `@if` so I was following the same style. Anyway, easy to change if  `$(REGSIZE)*8` is preferred.

Reply:
Nah, I didn't notice the other instruction, not worth changing if others are implemented with the `if`, and whoops, totally just went to the diff instead of reading pr message